spec/models/partner_spec.rb

```diff
@@ -100,6 +100,15 @@
     end
   end
 
+  describe '#invite_new_partner' do
```

Comment:
See comment above. I suggest testing the callback and checking to see if `PartnerUser.invite!` was called properly.

Reply:
Ha. Yeah, I definitely phoned it in on this. Updated it.